### ISSUE TITLE
NGI Project: PeerTube

### ISSUE THREAD

**Issue by fricklerhandwerk** [ngi-nix/ngipkgs#8]
PeerTube is a free, libre and federated video platform.

- [NLnet page](https://nlnet.nl/project/PeerTube/)
- [Peertube can be configured on NixOS](https://search.nixos.org/options?channel=unstable&query=peertube), but plugin support is not finished
- previous work:
  - https://github.com/NixOS/nixpkgs/pull/191497
  - https://github.com/ngi-nix/ngi/issues/102

The goal here is to provide infrastructure to add plugins to the NixOS module and get to work [`vosk`](https://gitlab.com/apps_education/peertube/plugin-transcription) and [`livechat`](https://github.com/JohnXLivingston/peertube-plugin-livechat).

Break down into separate issues as needed.

**Comment by cleeyv** [comment]
Other than the very broad category of both being activitypub-based web apps, GoToSocial and Peertube don't seem to have any overlap as projects, so I don't think they should be combined in a single issue like this. 

I was going to propose that GoToSocial be moved to its own issue, but when I looked into it further I found that as of the past month it is already fully packaged and has a module on nixpkgs: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/web-apps/gotosocial.nix 

So it seems like the simple solution is just to remove the GoToSocial parts of this issue. Any further thoughts on this, @fricklerhandwerk ?

**Comment by fricklerhandwerk** [comment]
@cleeyv Yeah, that was a typo on my end. I was preparing a GoToSocial issue and realised the same thing, and apparently failed to clean up the text. I'll edit the issue description.
